features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/package.json

```diff
@@ -15,17 +15,20 @@
   "author": "WSO2 Org",
   "license": "Apache License 2.0",
   "dependencies": {
-    "antd": "^2.11.2",
+    "antd": "^2.13.2",
```

Comment:
Do we need to update ant version?

Reply:
Revert with https://github.com/wso2/carbon-apimgt/pull/4890/commits/32e120551eb5cd6b154f8eb65ad7c3683c239716